chatkit-core/build.gradle

```diff
@@ -30,9 +30,6 @@ dependencies {
     testImplementation "org.spekframework.spek2:spek-dsl-jvm:$spek_version"
     testImplementation "org.spekframework.spek2:spek-runner-junit5:$spek_version"
 
-    testImplementation "org.jetbrains.spek:spek-api:$legacy_spek_version"
-    testImplementation "org.jetbrains.spek:spek-junit-platform-engine:$legacy_spek_version"
-
```

Comment:
on L42 it can be now just: `includeEngines 'spek2'`

Reply:
Good spot! Have updated 👍